sdk/storage/inc/datalake/datalake_options.hpp

```diff
@@ -286,17 +273,17 @@ namespace Azure { namespace Storage { namespace DataLake {
     /**
      * @brief The lease ID must be specified if there is an active lease.
      */
-    std::string LeaseId;
+    Azure::Core::Nullable<std::string> LeaseId;
```

Comment:
https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/storage/inc/common/access_conditions.hpp

We have `LeaseAccessConditions`

Reply:
Integrated access condition.